app/models/post.rb

```diff
@@ -1,2 +1,16 @@
 class Post < ApplicationRecord
+  belongs_to :author, class_name: 'User'
+  has_many :likes, class_name: 'Like'
+  has_many :comments, class_name: 'Comment'
```

Comment:
- [OPTIONAL] Just as discussed above. You may work with the suggestions below:

  ```suggestion
  has_many :likes
  has_many :comments
  ```

Reply:
This issue is well resolved! ✔️